packages/admin/dashboard/src/i18n/translations/fr.json

```diff
@@ -0,0 +1,2748 @@
+			"description": "Recherchez dans l'ensemble de votre boutique, y compris les commandes, les produits, les clients et plus encore.",
+			"allAreas": "Toutes les zones",
+			"navigation": "Navigation",
+			"openResult": "Ouvrir le résultat",
+			"showMore": "Voir plus",
+			"placeholder": "Accéder à ou rechercher n'importe quoi...",
+			"noResultsTitle": "Aucun résultat trouvé",
+			"noResultsMessage": "Nous n'avons trouvé aucun élément correspondant à votre recherche.",
+			"emptySearchTitle": "Tapez pour rechercher",
+			"emptySearchMessage": "Entrez un mot-clé ou une phrase pour explorer.",
+			"loadMore": "Charger {{count}} de plus",
+			"groups": {
+				"all": "Toutes les zones",
+				"customer": "Clients",
+				"customerGroup": "Groupes de clients",
+				"product": "Produits",
+				"productVariant": "Variantes de produits",
+				"inventory": "Inventaire",
+				"reservation": "Réservations",
+				"category": "Catégories",
+				"collection": "Collections",
+				"order": "Commandes",
+				"promotion": "Promotions",
+				"campaign": "Campagnes",
+				"priceList": "Listes de prix",
+				"user": "Utilisateurs",
+				"region": "Régions",
+				"taxRegion": "Régions fiscales",
+				"returnReason": "Motifs de retour",
+				"salesChannel": "Canaux de vente",
+				"productType": "Types de produits",
+				"productTag": "Tags de produits",
+				"location": "Emplacements",
+				"shippingProfile": "Profils d'expédition",
+				"publishableApiKey": "Clés API publiques",
+				"secretApiKey": "Clés API secrètes",
+				"command": "Commandes",
+				"navigation": "Navigation"
+			}
+		},
+		"keyboardShortcuts": {
+			"pageShortcut": "Sauter à",
+			"settingShortcut": "Paramètres",
+			"commandShortcut": "Commandes",
+			"then": "alors",
+			"navigation": {
+				"goToOrders": "Commandes",
+				"goToProducts": "Produits",
+				"goToCollections": "Collections",
+				"goToCategories": "Catégories",
+				"goToCustomers": "Clients",
+				"goToCustomerGroups": "Groupes de clients",
+				"goToInventory": "Inventaire",
+				"goToReservations": "Réservations",
+				"goToPriceLists": "Listes de prix",
+				"goToPromotions": "Promotions",
+				"goToCampaigns": "Campagnes"
+			},
+			"settings": {
+				"goToSettings": "Settings",
+				"goToStore": "Boutique",
+				"goToUsers": "Utilisateurs",
+				"goToRegions": "Régions",
+				"goToTaxRegions": "Régions fiscales",
+				"goToSalesChannels": "Canaux de vente",
+				"goToProductTypes": "Types de produits",
+				"goToLocations": "Emplacements",
+				"goToPublishableApiKeys": "Clés API publiques",
+				"goToSecretApiKeys": "Clés API secrètes",
+				"goToWorkflows": "Workflows",
+				"goToProfile": "Profil",
+				"goToReturnReasons": "Motifs de retour"
+			}
+		},
+		"menus": {
+			"user": {
+				"documentation": "Documentation",
+				"changelog": "Changelog",
+				"shortcuts": "Raccourcis",
+				"profileSettings": "Paramètres du profil",
+				"theme": {
+					"label": "Thème",
+					"dark": "Dark",
+					"light": "Light",
```

Comment:
Q: should we keep it in english or switch to `Sombre`/`Clair`?

Reply:
i would suggest to keep english as it's a common use or if we translate it keep the lowercase as it doesn't seems to need an Uppercase.